src/main/java/uk/nhs/digital/nhsconnect/nhais/configuration/AwsTrustStore.java

```diff
@@ -0,0 +1,89 @@
+package uk.nhs.digital.nhsconnect.nhais.configuration;
+
+import lombok.RequiredArgsConstructor;
+import lombok.SneakyThrows;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.TrustManagerFactory;
+import javax.net.ssl.X509TrustManager;
+import java.io.FileInputStream;
+import java.security.KeyStore;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+public class AwsTrustStore {
+
+    @SneakyThrows
+    public static void addToDefault(String trustStorePath, String trustStorePassword) {
+        final X509TrustManager defaultTrustManager = getDefaultTrustManager();
+        final X509TrustManager awsTrustManager = getAwsTrustManager(trustStorePath, trustStorePassword);
+        X509TrustManager combinedTrustManager = new CombinedTrustManager(awsTrustManager, defaultTrustManager);
+
+        SSLContext sslContext = SSLContext.getInstance("TLS");
+        sslContext.init(null, new TrustManager[]
```

Comment:
[optional] can you create a new var out of this new TrustManager[]?

Reply:
I think that wouldn't help